packages/config/src/class-config.php

```diff
@@ -78,6 +80,11 @@ public function on_plugins_loaded() {
 			$this->ensure_class( 'Automattic\Jetpack\Sync\Main' )
 				&& $this->ensure_feature( 'sync' );
 		}
+
+		if ( $this->config['jitm'] ) {
+			$this->ensure_class( 'Automattic\Jetpack\JITM' )
+				&& $this->ensure_feature( 'sync' );
```

Comment:
This should be `jitm`, not `sync`.
```suggestion
				&& $this->ensure_feature( 'jitm' );
```

Reply:
Whoops, copy-pasta! Thanks, good catch!